xbmc/music/Song.cpp

```diff
@@ -38,7 +39,33 @@ CSong::CSong(CFileItem& item)
   std::vector<std::string> musicBrainArtistHints = tag.GetMusicBrainzArtistHints();
   strArtistDesc = tag.GetArtistString();
   if (!tag.GetMusicBrainzArtistID().empty())
-  { // have musicbrainz artist info, so use it
+  { // Have musicbrainz artist info, so use it
+    // Establish tag consistency - do the number of musicbrainz ids and number of names in hints or artist match
```

Comment:
I wondered about that myself. Then thought a whole new unit just for that seemed silly and less readable. There are some differences in the code too, and a possibility of more, so I decided against it in the end.